packages/fhir-converter/src/lib/cda/cda.js

```diff
@@ -167,7 +169,28 @@ module.exports = class cda extends dataHandler {
           parseString(data, parseOptions, (err, result) => {
             if (err) {
               // if still throwing an error, give up
-              reject(err);
+              const parser = new XMLParser({
```

Comment:
let's raise a follow up issue and leave a TODO comment accordingly, so we don't forget to evaluate replacing the xml2js with fast-xml-parser

Reply:
https://github.com/metriport/metriport-internal/issues/2107

added to the master issue for this